charts/cinder-csi-plugin/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 appVersion: latest
-description: Cinder CSI Plugin for OpenStack
+description: Cinder CSI Chart for OpenStack
 name: openstack-cinder-csi
-version: 1.1.2
+version: 1.2.0
```

Comment:
Maybe I missed the discussion, but may I know where this version comes from? I see it's still 1.1.2 on master branch?

Reply:
It should be 1.2.0 on master now with #1190